### DISCUSSION TITLE
How to display local images with chakra-ui Image component

### DISCUSSION THREAD

**Discussion by gpmarchi** [chakra-ui/chakra-ui#5495] [Q&A]
Hi,

I'm using Next.js and I've been trying to use the chakra-ui `Image` component to display local images from my assets folder that resides in the root src folder.

So if I import the image from the assets folder and try to use it in the component like this:

```typescript
...

import logo from '../../assets/logo.svg';

...

<Image src={logo} />

```

I don't get any errors but the image doesn't get displayed correctly.

Now if I replace the chakra `Image` component with the native one from Next, everything works and the image is displayed correctly.

I'd like to use chakra-ui component instead for the ease of styling.

How can I load and display local images from my project using chakra-ui?

**Comment by ahmedllshafiey** [comment]
You must write local URL from public like '/images/logo.svg'
the above code works with next/image not chakra Ui
